cleanlab/filter.py

```diff
@@ -290,6 +302,26 @@ class 0, 1, ..., K-1.
             multi_label=multi_label,
             return_indices_of_off_diagonals=True,
         )
+
+    if filter_by in ["low_normalized_margin", "low_self_confidence"]:
+        # TODO: consider setting adjust_pred_probs to true based on benchmarks (or adding it kwargs, or ignoring and leaving as false by default)
+        scores = get_label_quality_scores(
```

Comment:
Can you add unit test to ensure this gets covered?

Reply:
I don't actually know why codecov thinks this line is not covered, I have added multiple unittests that should call this line (ref [here](https://github.com/cleanlab/cleanlab/pull/595/files/e83d7ea4b1bd19ae626b0134db598cce5d2c41e2#diff-817ca800504a08d514a20253405df1f597ee171107cc9e47e564fdacd68f0473R1014)), also the lines below are covered (and they must access the variable `scores`, ie. cannot be covered unless this line is called), so I'm a little confused.

Reply:
no worries lets ignore this then